planner/type_join.go

```diff
@@ -545,12 +555,23 @@ func (n *typeJoinMany) Next() (bool, error) {
 	if n.index != nil {
 		// @todo: handle index for one-to-many setup
 	} else {
+		// filterRaw := request.Filter{
+		// 	Conditions: map[string]any{
+		// 		n.rootName + "_id": map[string]any{
+		// 			"_eq": n.currentValue.GetKey(),
+		// 		},
+		// 	},
+		// }
+		// filter := mapper.ToFilter(filterRaw, n.subType.DocumentMap())
 		fkIndex := &mapper.PropertyIndex{
 			Index: n.subSelect.FirstIndexOfName(n.rootName + "_id"),
 		}
 		filter := map[connor.FilterKey]any{
-			fkIndex: n.currentValue.GetKey(), // user_id: "bae-ALICE" |  user_id: "bae-CHARLIE"
+			fkIndex: map[connor.FilterKey]any{
+				mapper.FilterEqOp: n.currentValue.GetKey(), // user_id: "bae-ALICE" |  user_id: "bae-CHARLIE"
```

Comment:
question: Is the comment still relevant? I can't see how it helps understanding what's happening.

Reply:
no idea, thought it was weird when I was making the change. Will remove